app-backend/common/src/main/scala/KamonTrace.scala

```diff
@@ -7,7 +7,7 @@ import kamon.trace.Tracer
 import scala.concurrent.Future
 import scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
May be to use `def timedFuture[T](name: String)(future: Future[T])(implicit ec: ExecutionContext)` signature instead of this? // however it doesn't matter that much